.circleci/config.yml

```diff
@@ -70,18 +70,52 @@ jobs:
           at: ./
       - run: npm run lint
 
-  build:
+  build-branch-en:
     docker: *node_image
     steps:
       - attach_workspace:
           at: ./
-      - run: npx ng build --configuration ci --base-href //d39se6ub293skr.cloudfront.net/branch/<< pipeline.git.branch >> --progress=false
+      - run: npx ng build --configuration production-en --base-href /branch/<< pipeline.git.branch >>/ --deploy-url //d39se6ub293skr.cloudfront.net/branch/<< pipeline.git.branch >>/en/ --progress=false
       - run: echo '<html><head><script>window.location.replace("en/");</script></head></html>' > dist/algorea/index.html
       - persist_to_workspace:
           root: ./
           paths:
             - dist/algorea
 
+  build-branch-fr:
+    docker: *node_image
+    steps:
+      - attach_workspace:
+          at: ./
+      - run: npx ng build --configuration production-fr --base-href /branch/<< pipeline.git.branch >>/ --deploy-url //d39se6ub293skr.cloudfront.net/branch/<< pipeline.git.branch >>/fr/ --progress=false
+      - persist_to_workspace:
+          root: ./
+          paths:
+            - dist/algorea
+
+  build-master-en:
+    docker: *node_image
+    steps:
+      - attach_workspace:
+          at: ./
+      - run: npx ng build --configuration production-en --base-href / --deploy-url //d39se6ub293skr.cloudfront.net/en/ --progress=false
+      - run: echo '<html><head><script>window.location.replace("en/");</script></head></html>' > dist/algorea/index.html
+      - persist_to_workspace:
+          root: ./
+          paths:
+            - dist/algorea
+
+  build-master-fr:
+    docker: *node_image
+    steps:
+      - attach_workspace:
+          at: ./
+      - run: npx ng build --configuration production-fr --base-href / --deploy-url //d39se6ub293skr.cloudfront.net/fr/ --progress=false
+      - persist_to_workspace:
```

Comment:
I think I would prefer defining a command (kind of function) to prevent repeating the same again and again.